### ISSUE TITLE
Add support for scatter with multiple values for each point.

### ISSUE THREAD

**Issue by t20100** [silx-kit/silx#3274]
A sort of `StackScatterView`, similar to `ScatterView` but with support for multiple values defined for each point.
First implementation can be to have a slider to scroll through the different values, but compositing multiple values (either as colors or transparency channels) would also be great.
Not sure whether it is a separated widgets (probably) or `ScatterView` can be extended.

**Comment by vasole** [comment]
I guess one will need a table to handle the items (associated single color colormap, shown/hidden, alpha)

**Comment by vallsv** [comment]
I dont exactly know your usecase, but be careful that you also could have a stack of scatters. Which means not only many values, but also many positions. That's the case for continuous scans in 3D (so many scatters not taken at the same time).

**Comment by vasole** [comment]
The use case is multiple scalars collected at the same X and Y positions.

**Comment by t20100** [comment]
in NeXus, that would be some `auxiliary_signals` and no auxiliary axes if that ever exits.

**Comment by vasole** [comment]
No, you have to assume that all the auxiliary signals share the same coordinates.

I wanted to have auxiliary_axes but they did not agree (the use case being raw data and a fit of a region).
